lib/components/app/app.css

```diff
@@ -84,3 +84,12 @@
 .view-switcher button.btn-link:hover {
   color: #fff;
 }
+
+/* Full screen modal styling */
+.fullscreen-modal {
+  width: 90vw;
+  height: 100vh;
```

Comment:
Replace this one with the `90vh` defined below, and remove that subcontent CSS.

Reply:
Then you can add the CSS for the iframe element here too.

Reply:
`.fullscreen-modal` is the dark background, `.fullscreen-modal .modal-content` is the actual modal. I don't see how these rules can be collapsed into one?

Reply:
Oh really, then why would you want to override the size of the dark background?